src/main/java/seedu/address/model/applicant/Id.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.model.applicant;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents an Applicant's Id in the address book.
+ */
+public class Id {
+
+    public static final String MESSAGE_CONSTRAINTS = "Id can take any values, and it should not be blank";
+
+    /*
+     * The first character of the Id must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[^\\s].*";
+
+    public final String value;
+
+    /**
+     * Constructs an Id object.
+     *
+     * @param id A valid Id string
+     */
+    public Id(String id) {
```

Comment:
Constructor should take in an int id.

Reply:
Noted, have corrected and changed the id value to be an Integer, and changed the respective methods.